<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Form</class>
 <widget class="QWidget" name="Form">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>748</width>
    <height>653</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>KILL THE FATBIES</string>
  </property>
  <widget class="QPushButton" name="boton_inicio">
   <property name="geometry">
    <rect>
     <x>130</x>
     <y>510</y>
     <width>501</width>
     <height>81</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>NanumGothic</family>
     <pointsize>20</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>INICIAR PARTIDA</string>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>220</x>
     <y>50</y>
     <width>491</width>
     <height>61</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>NanumMyeongjo</family>
     <pointsize>28</pointsize>
     <weight>75</weight>
     <italic>false</italic>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>WELCOME TO</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>90</x>
     <y>120</y>
     <width>581</width>
     <height>51</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>NanumMyeongjo</family>
     <pointsize>48</pointsize>
     <weight>75</weight>
     <italic>false</italic>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>KILL THE FATBIES</string>
   </property>
  </widget>
  <widget class="QLabel" name="zombie">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>190</y>
     <width>281</width>
     <height>291</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="zombie_2">
   <property name="geometry">
    <rect>
     <x>220</x>
     <y>190</y>
     <width>281</width>
     <height>291</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="zombie_3">
   <property name="geometry">
    <rect>
     <x>430</x>
     <y>200</y>
     <width>281</width>
     <height>291</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <zorder>zombie_3</zorder>
  <zorder>zombie</zorder>
  <zorder>boton_inicio</zorder>
  <zorder>label</zorder>
  <zorder>label_2</zorder>
  <zorder>zombie_2</zorder>
 </widget>
 <resources/>
 <connections/>
</ui>
